packages/lesswrong/components/comments/PopularCommentsList.tsx

```diff
@@ -24,15 +25,17 @@ const PopularCommentsList = ({classes}: {classes: ClassesType}) => {
 
   const {LoadMore, PopularComment} = Components;
   return (
-    <div className={classes.root}>
-      {results?.map((comment) =>
-        <PopularComment
-          key={comment._id}
-          comment={comment}
-        />
-      )}
-      <LoadMore {...loadMoreProps} />
-    </div>
+    <AnalyticsContext listContext="popularCommentsList">
```

Comment:
Looks like `listContext` is deprecated (see `analyticsEvents.tsx`). Since the `ExpandableSection` around this has a `pageSectionContext` value already, do we still need to add a context here?

Reply:
Good catch - wasn't aware of this!

I've changed it to use `pageSection` instead. I think it's worth keeping the context here as I don't want to rely on this always being inside that other context in the indefinite future.